tf2_ros/src/tf2_ros/buffer.py

```diff
@@ -33,6 +33,8 @@
 from tf2_msgs.srv import FrameGraph, FrameGraphResponse
 import rosgraph.masterapi
 
+CAN_TRANSFORM_POLLING_SCALE = 0.01
```

Comment:
I liked having the TRANSFORM_POLLING_SCALE as a member variable. This gives the user a way to control it if they do care about it for a specific use case/module.

Please update this to use the global on for a `DEFAULT_CAN_TRANSFORM_POLLING_SCALE` which then will be used in the `__init__`  to initialize `self.CAN_TRANSFORM_POLLING_SCALE`

This will avoid magic numbers in the class, but also mean that the developer doesn't have to think about overriding a global.